tests/instance/test_kubernetes.py

```diff
@@ -418,6 +419,7 @@ def test_statefulset(
             "ready_replicas": 1,
             "create_timestamp": datetime.datetime(2021, 4, 1).timestamp(),
             "git_sha": "aaa000",
+            "image_version": None,
```

Comment:
would be nice to see a test for the case where the label is set

Reply:
I've added a new one that's pretty much the same as `test_statefulset`, I wasn't sure if I should just set the image_version in `test_statefulset` or if we wanted a new test, so went for the latter, happy to change it though